sdk/tests/conformance2/rendering/00_test_list.txt

```diff
@@ -18,5 +18,6 @@ framebuffer-unsupported.html
 instanced-arrays.html
 --min-version 2.0.1 instanced-rendering-bug.html
 out-of-bounds-index-buffers-after-copying.html
+rendering-sampling-feedback-loop.html
```

Comment:
We're trying to wrap up version 2.0.0 of the conformance suite. At this point I think we should put pretty much all new tests except for those for the texture sub-source entry points into 2.0.1. What do you think @Richard-Yunchao @zhenyao @jdashg ?